downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,64 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ## Backward-incompatible changes
 
+- A type-checking bug that resulted in incorrect subtype relationships between records with optional fields and open records has been fixed.
```

Comment:
```suggestion
- Fixed a type-checking bug that resulted in incorrect subtype relationships between records with optional fields and open records.
```

Reply:
Again, please see https://github.com/ballerina-platform/ballerina-dev-website/pull/7642#discussion_r1305092375. Rest is for consistency.